src/data/entity_registry.ts

```diff
@@ -27,6 +27,16 @@ export interface EntityRegistryEntryUpdateParams {
   new_entity_id?: string;
 }
 
+export const findBatteryEntity = (
```

Comment:
Should be `sensor` or `binary_sensor` domain. I wonder, should we add `device_class` to `ExtEntityRegistryEntry` and we don't need states? We already have that data in the backend in the entity registry.

Reply:
Yeah, that would be nice, I'll do it in a followup PR.